tun_tap/restart_modem_gui.sh

```diff
@@ -75,5 +75,5 @@ cat /sys/kernel/debug/iio/iio:device3/direct_reg_access
 
 
 /usr/local/bin/modemd -a $ip -m $subnet -d $delay & 
-#/usr/local/bin/en_macsec.sh
+/usr/local/bin/en_macsec.sh
```

Comment:
Same - I don't think this has to do with the fragmentation?

Reply:
It does not have to do with fragmentation but I wanted to enable encryption now that the TAP should work